### ISSUE TITLE
autotag creations with "case_sensitive = false" fails

### ISSUE THREAD

**Issue by ARZ-SagA** [dynatrace-oss/terraform-provider-dynatrace#5]
Hi,

like in #3 the modfying or creating a autotag ressource fails if the case_sensitive attribute is false (if the string comparison operator is "BEGINS_WITH)"

Sample Ressource
```
resource "dynatrace_autotag" "stage" {
	rules {
		conditions {
			string_comparison {
				negate = false
				type = "STRING"
				operator = "BEGINS_WITH"
				value = "p"
				case_sensitive = false
			}
			base_condition_key {
				attribute = "HOST_GROUP_NAME"
			}
		}
		enabled = true
		propagation_types = [ "PROCESS_GROUP_TO_HOST", "PROCESS_GROUP_TO_SERVICE" ]
		type = "PROCESS_GROUP"
		value_format = "p"
	}
	name = "Stage"
}
```

Output on Create:
```
dynatrace_autotag.stage: Creating...

Error: rpc error: code = Unavailable desc = transport is closing
```

Output on Modify:
```
dynatrace_autotag.stage: Modifying... [id=74af98f9-5067-404b-be11-696a8b16b462]

Error: {
  "code": 400,
  "message": "Constraints violated.",
  "constraintViolations": [
    {
      "parameterLocation": "PAYLOAD_BODY",
      "location": "",
      "message": "caseSensitive must be null if the operator 'EXISTS' is specified, for any other operator it must be set.",
      "path": "rules[0].conditions[0].comparisonInfo.caseSensitive"
    }
  ]
}
```

Thanks and best regards,

Alex

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Seems to have the same root cause as #3.
The OpenAPI Spec of the Config REST API declares `caseSensitive` as optional, while there are payloads where it isn't.
We've already filed a bug report to Dynatrace R&D for the flawed OpenAPI Spec.

Nevertheless we'll introduce the same workaraound as the one in #3 with v1.0.5.
Thanks!

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Fixed with [v1.0.5](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/tag/v1.0.5)
